charts/prometheus-stackdriver-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Stackdriver exporter for Prometheus
 name: prometheus-stackdriver-exporter
-version: 4.0.0
+version: 4.1.0
```

As this seems only to be a Bugfix:

```suggestion
version: 4.0.1
```